functions/memorystore/redis/redis.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START functions_memorystore_redis]
+
+// Package redis provides a basic Cloud Function that connects
+// to a managed Redis instance.
+package redis
```

Comment:
Sorry for the package name churn. Better to have it not match the directory name and be something other than `redis` since you're importing a package named `redis` below. Things get confusing with collisions like that.

Reply:
No worries, done :+1: